File: AddIns/Templates/VBServerPackage/Component.vb
Public Class Component
    Inherits Srvtools.DataModule
    Public Sub New(ByVal container As System.ComponentModel.IContainer)
        container.Add(Me)
        InitializeComponent()
    End Sub 'New
    Public Sub New()
        InitializeComponent()
    End Sub 'New
    Private Sub InitializeComponent()
        Me.components = New System.ComponentModel.Container
        Me.ServiceManager = New Srvtools.ServiceManager(Me.components)

    End Sub
    Private components As System.ComponentModel.IContainer
    Friend WithEvents ServiceManager As Srvtools.ServiceManager
End Class

File: CBBN/sRT102/Component.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Diagnostics;
using System.Linq;
using System.Text;
using Srvtools;
using Newtonsoft;
using Newtonsoft.Json;
using System.Web;
using System.Data;

namespace sRT102
{
    public partial class Component : DataModule
    {
        public Component()
        {
            InitializeComponent();
        }

        public Component(IContainer container)
        {
            container.Add(this);

            InitializeComponent();
        }

        public object[] smRT1021(object[] objParam)
        {
            var ss = (string)objParam[0];
            var sdata = ss.Split(',');
            //開啟資料連接
            IDbConnection conn = ucRT1021.Connection;
            conn.Open();
            //設定輸入參數的值
            try
            {
                ucRT1021.InfoParameters[0].Value = sdata[0];
                ucRT1021.InfoParameters[1].Value = sdata[1];
                ucRT1021.InfoParameters[2].Value = sdata[2];
                /*取得統計的結果，並將結果返回*/
                double ii = ucRT1021.ExecuteNonQuery();
                return new object[] { 0, ii };
            }
            catch (Exception ex)
            {
                return new object[] { 0, ex };
            }

        }
    }
}
